packages/ibm-products/src/components/NotificationsPanel/NotificationsPanel_data.js

```diff
@@ -39,6 +39,7 @@ const data = [
     title: 'System alert',
     description: 'Email classification was exported successfully.',
     timestamp: new Date(currentDate.getTime() - 11 * msInOneMinute),
+    unread: false,
```

Comment:
Added back missing `unread:false` key/value. Expected keys should always be included even if their value is false or null.